### ISSUE TITLE
Figure out how to work with GlobalSymbols

### ISSUE THREAD

**Issue by SteveALee** [mulberrysymbols/mulberry-symbols#30]
The [GlobalSymbols](https://globalsymbols.com/) project has objectives like unifying and localising symbols across sets. We should figure out how to best work with them.

**Comment by SteveALee** [comment]
Recently Global symbols added BoardBuilder and a translation tool. There is also work on vocabs and tooling for translation. Russel also asked how he should get the symbols (currently using  symbol-info CSV)

See my [blog post](https://opendirective.net/2020/04/17/the-open-communication-symbols-ecosystem/)

So eventually we could provide an API for notifiying of and supplying symbol updates to GlobalSymbols. Translation could be provided by Global Symbols. Users can use BordBuilder to then create boards for printing or export to AAC via the OBF. BoardBuild can access the symbols via GlobalSymbols cache. We'd need t ofigure how to make symbols available for reference in OBF/Z files and apps using it.

**Comment by neilt1700** [comment]
> See my [blog post](https://opendirective.net/2020/04/17/the-open-communication-symbols-ecosystem/)

There seems to be a problem with that link - SSL connection doesn't seem to be working properly.

**Comment by SteveALee** [comment]
@neilt1700 you're the second person to have that issues. But it IS all configured correctly. It's served on GitHub Pages after all. Unless I can reproduce I'm a bit stuck :(

I this what you see
<pre>
This site can’t provide a secure connection

opendirective.net uses an unsupported protocol.

ERR_SSL_VERSION_OR_CIPHER_MISMATCH

Unsupported protocol

The client and server don't support a common SSL protocol version or cipher suite.

 </pre>

This might help - https://kinsta.com/knowledgebase/err_ssl_version_or_cipher_mismatch/

Steve

**Comment by neilt1700** [comment]
Yes that's the error. Using BT broadband - it fails with that error. Same problem on all browsers, however no error if I use the Tor Browser (still BT broadband). If I connect using a different Wi-Fi (Three) I don't get any errors from any browser.

**Comment by SteveALee** [comment]
How bizarre! I'm on BT too (but FtP) and going via an extension disc with very latest HomeHub. I blame their firewall thingie. I raised a support request with Github anyway. I'll see if other user is also on BT

**Comment by eadraffan** [comment]
No problems with Plusnet (using FF, Chrome or Edge).  The company is owned by BT, but different organisation.

**Comment by SteveALee** [comment]
Seems to be a [long standing problem](https://community.bt.com/t5/BT-Fibre-broadband/quot-ERR-SSL-VERSION-OR-CIPHER-MISMATCH-Unsupported-protocol/m-p/1999167#M297832
) with their family protection filters giving a rubbish error for blocked sites.

Now I have to figure how to get white listed! Pathetic really.

**Comment by neilt1700** [comment]
Just wondering if it could be a DNS issue - could you try using a different domain name server? (we use DNS Made Easy - started using when we were on 123-REG a few years ago because of problems we had with them - including site not being available everywhere). Alternatively are you able to try serving it from a different host provider with a different IP address? Might rule out another potential source of the problem.

**Comment by SteveALee** [comment]
Interesting idea! DNS is indeed by 123reg but it's hosted on GitHub pages. 

https://185.199.108.153/ errors with `NET::ERR_CERT_COMMON_NAME_INVALID`. but that might be GitHub. https://tools.dnsstuff.com/#dnsReport|type=domain&&value=opendirective.net takes forever, and that maybe a bad smell

I'll rummage a bit more in the DNS settings. 

Maybe I'll move everything over to the excellent Krystal. Thanks
